pandas/core/generic.py

```diff
@@ -5075,9 +5075,9 @@ def pipe(self, func, *args, **kwargs):
         Notes
         -----
         - `agg` is an alias for `aggregate`. Use the alias.
-        - Some NumPy functions resolve to their corresponding internal Cython function. As pandas operations
-          generally exclude NaNs, this means that `.agg(np.nanmedian)`, `.agg(np.median)`, and `.agg('median')
-          will return the same result.
+        - Some NumPy functions resolve to their corresponding internal Cython function.
+          As pandas operations generally exclude NaNs, for example `.agg(np.nanmedian)`,
+          `.agg(np.median)`, and `.agg('median') will return the same result.
```

Comment:
Cool, IMO this documents the behaviour nicely without being too long. Just a suggestion on the wording (probably needs reformatting) 
```
For example, `.agg(np.nanmedian)`, `.agg(np.median)`, and `.agg('median') all return the same result (note that pandas operations generally exclude NaNs).
```
if after this you ping a core member (e.g. datapythonista) I'd like to think it'd get merged relatively quickly (but there's _lots_ of open issues, so please be patient if it takes some time)

Reply:
@MarcoGorelli Yes, perfect. Will update with changes. Should I update `transform` docstring too? I'll keep the same wording but replace `.agg` with `.transform`. 

Yes, will ping him once PR passes all the checks. Thank you.